src/main/java/team/creative/creativecore/common/util/type/list/Tuple.java

```diff
@@ -36,12 +36,12 @@ public int hashCode() {
     public boolean is(K key) {
         if (this.key != null)
             return this.key.equals(key);
-        return this.key == key;
+        return key == null;
     }
     
     @Override
     public boolean equals(Object obj) {
-        if (obj instanceof Tuple tuple)
+        if (obj instanceof Tuple<?, ?> tuple)

```

Comment:
Please avoid these. That's the style of code of this project and it does not make sense to do it different in a few locations.

Reply:
done